Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -583,6 +583,9 @@ class SearchViewController: SiteTableViewController,
                   let url = defaultEngine.searchURLForQuery(suggestion)
             else { return }
 
+            // Ecosia: increment search count when bing distribution
+            BingDistributionExperiment.incrementCounterIfTestVariant()
```

Comment:
We had two places where the increment should happen, one was this one and then the other was `BrowserViewController+URLBarDelegate` as shown in [the PR](https://github.com/ecosia/ios-browser/pull/650/files).

I didn't see this second part here though, did this change with the upgrade somehow?

Reply:
Awesome! The cherry pick didn't show this as a conflict for some reason but I added it after double checking your comment.